cmd/kogito/command/install/data_index.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 const (
+	defaultDataIndexHTTPPort             = 8080
```

Comment:
We already have this one set on the `controller` package. What you think of moving it to `infrastructure` package?

Reply:
agreed.